### PR TITLE
Add possibility to exclude certain parts from the build

### LINKED ISSUE #12: Include icons optional
We should allow users to use ember-uikit without icons, since they increase the payload a lot.

### PR DESCRIPTION
**Pull request by anehx**
This PR adds some options and fixes #12:

```javascript
'ember-uikit': {
  importUIkitCSS: true, // excludes css from build if false
  importUIkitJS: true, // excludes js from build if false
  importUIkitAssets: true, // excludes assets (not icons) from build if false
  importUIkitIcons: true, // excludes icons from build if false

  useIcons: true, // excludes all icon related stuff if false

  whitelist: [], // components which will be included
  blacklist: [] // components which will be excluded
}
```

**TODO**

- [x] Document options
- [x] Exclude `uk-icon` if `useIcons: false`

### CODE REVIEW

**Comment by jayvarner** [comment]
The treeshake pattern is very elegant. Documentation seems clear. Nice work!

File: tests/dummy/app/templates/index.hbs [hunk 1]
```diff
@@ -22,6 +22,76 @@
...
+      <td>Whether to use UIkits icons</td>
+    </tr>
+    <tr>
+      <td><code>whitelist</code></td>
+      <td><code>String[]</code></td>
+      <td><code>[]</code></td>
+      <td>
+        A list of included components. Only components in this list will be
+        included in your build. You should never use <code>whitelist</code>
+        and <code>blacklist</code>!
```
Comment (czosel) [inline tests/dummy/app/templates/index.hbs]: If you say that the two lists should not be used at the same time, i'd expect an error to be thrown in that case. I think right now the blacklist takes precedence.

Comment (anehx) [reply tests/dummy/app/templates/index.hbs]: That should definitely throw an error!

Comment (anehx) [reply tests/dummy/app/templates/index.hbs]: Or at least a warning

**Pull request merged**